idb/postgres/postgres_integration_common_test.go

```diff
@@ -17,8 +18,24 @@ import (
 	"github.com/algorand/indexer/util/test"
 )
 
+var testpg = flag.String("test-pg", "", "postgres connection string")
+
 // setupPostgres starts a gnomock postgres DB then returns the connection string and a shutdown function.
 func setupPostgres(t *testing.T) (*sql.DB, string, func()) {
+	if testpg != nil && *testpg != "" {
+		// TODO: Drop schema?
```

Comment:
Without dropping the schema, this will not work for multiple tests, right?

Reply:
Yes exactly. This comment is my only contribution to Brian's work here.

Reply:
I'm not sure what you are referring to. In any case, we need to drop the schema. Otherwise, the tests might not do what they are supposed to do!

Reply:
Setup is an external problem. I'd presume that whatever is handing us a link to a database just ran `createdb` or otherwise ensured the db was empty and ready to use.

Reply:
That's a reasonable assumption / requirement, but not sufficient if you are executing multiple tests.